### DISCUSSION TITLE
Hi, I'd like to ask how to send a message via telegram's @username

### DISCUSSION THREAD

**Discussion by qianmianyao** [gotd/td#1005] [Q&A] [answered]
Hi, I would like to ask how to send a message via telegram @username, is there any documentation or demo about it?

**Comment by ernado** [comment, marked as answer]
[telegram/message#Sender.Resolve](https://pkg.go.dev/github.com/gotd/td@v0.79.0/telegram/message#Sender.Resolve)

https://github.com/gotd/td/blob/adfab92211b96e4f389d559888007bc962c70791/examples/userbot/main.go#L181-L187
